modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -22,21 +23,46 @@ Hot plug a bridge network interface to a virtual machine (VM) while the VM is ru
 $ virtctl start <vm_name>
 ----
 
-. Use the following command to hot plug a new network interface to the running VM. The `virtctl addinterface` command adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM.
+. Add the new network interface to the running VM by editing the VM specification. This adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM.
 +
-[source,terminal]
+.Example VM configuration
+[source,yaml]
 ----
-$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+template:
+  spec:
+    domain:
+      devices:
+        interfaces:
+        - name: defaultnetwork
+          masquerade: {}
+          # new interface
+        - name: <secondary_nic> <1>
+          bridge: {}
+    networks:
+    - name: defaultnetwork
+      pod: {}
+      # new network
+    - name: <secondary_nic> <2>
+      multus:
+        networkName: <nad_name> <3>
+# ...
 ----
-+
-where:
-
-<vm_name>:: Specifies the name of the `VirtualMachine` object.
-<net_attach_def_name>::  Specifies the name of `NetworkAttachmentDefinition` object.
-<interface_name>:: Specifies the name of the new network interface.
+<1> Specifies the name of the new network interface.
+<2> Specifies the name of the network. This must be the same as the value of the `template.spec.domain.devices.interfaces.name` field. 
+<3> Specifies the name of `NetworkAttachmentDefinition` object. 
 
+. Apply the VM configuration:
```

Comment:
This is wrong.
The VM manifest was already applied, and the VM is running at this point.
The changes (the new interface) should be added via `oc edit vm`. I don't think that the VMI is updated this way.
@AlonaKaplan Please keep me honest here. Thank you.

Reply:
@AlonaKaplan Can you please respond?

Reply:
`oc apply -f` will work, but indeed it is better to use `oc edit`.

Reply:
It didn't work for me, even though I created the VM with `oc apply -f`.

Reply:
Could you please share the error you got?

Reply:
I have updated the docs to use `oc edit`

Reply:
> Could you please share the error you got?

I didn't see any error, just that the VMI spec was not updated, thus the guest VM was not updated as well (after migration).